ynr/apps/candidates/templatetags/standing.py

```diff
@@ -68,6 +67,12 @@ def post_in_election(person, election):
             )
         prefix, suffix = get_known_candidacy_prefix_and_suffix(candidacy)
         result = prefix + result + suffix
+        if candidacy.previous_party_affiliations.exists():
+            for party in candidacy.previous_party_affiliations.all():
+                result += '<span class="previous-party"><li>Previous party affiliation: {}</li></span>'.format(
+                    party.name
+                )
+
```

Comment:
@michaeljcollinsuk Based on your other comments, I'm not sure this is right? It should be based on the candidate, not the candidacy (aka membership)?